### ISSUE TITLE
Add option to specify per-file or per-group processors in wro.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [wro4j/wro4j#856]
```
Right now, we have to have a global list of processors in wro.properties which 
is applied to all groups.

It would be nice to have extra attributes in wro.xml to specify per-file or 
per-group processors, for example:

<group name="all" preProcessors="cssImport" postProcessors="cssCompressor">
  ...
</group>

I'm interested in this because I have some already minimized files that I want 
to concatenate with the other files, but I don't want to process them in any 
way otherwise.

```

Original issue reported on code.google.com by `img...@gmail.com` on 3 Mar 2014 at 6:59

**Comment by detollenaere** [comment]
I'm really interested by this feature because it's common to apply specific strategies regarding the files or groups. 

Hope this feature will come soon ;)

**Comment by bgillis** [comment]
+1

**Comment by olkainfry** [comment]
+1

**Comment by Anthares** [comment]
Hi,

First many thanks for the work done on this awesome project :)

I might be digging out something really out, but I figured out I need something like this proposal, with some modifications.

I'm currently working on a new project using Angular, Spring Boot, Wro4J and [Wro4J Spring Boot Starter](https://github.com/michael-simons/wro4j-spring-boot-starter).
I've already used Wro4J on other projects where I used Wro4J only on my code, but now I want to include all my libs.

I've split resources in four groups, minified groups for production, and non-minified for development.
- minified libs
- minified app
- non-minified libs
- non-minified app


I ended up feeling frustrated seeing I can't specify different pre/post-processors on libs I'm using and on my own app.
Most libs comes with an already minified version. So I just want to merge libs in a single file with almost no processing, while my app in heavily minified and merged in another file.

So I started monkeying in Wro's source code and reading issues here.

What I think is to allow to specify different pre/post-processor on groups through properties.
I think WroModel should stay unchanged, as its purpose is to define groups and resources.

The idea is to allow to specify group and processors in properties, something like the following.
Ex. :
```xml
<?xml version="1.0" encoding="UTF-8"?>
<groups xmlns="http://www.isdc.ro/wro">
  <group name='core'>
    <css>/static/libs/core/core.css</css>
    <js>/static/libs/core/core.js</js>
  </group>
  <group name='other-lib'>
    <css>/static/libs/other/other.css</css>
    <css>/static/libs/other/other-extensions.css</css>
    <js>/static/libs/other/other.js</js>
  </group>
</groups>
```
```Properties
preProcessors=<some-pre-processors>
postProcessors=<some-post-processors>
groups.1.name=core
groups.1.preProcessors=<other-pre-processors>
groups.1.postProcessors=<other-post-processors>
groups.2.name=other-lib
groups.2.preProcessors=<more-pre-processors>
groups.2.postProcessors=<more-post-processors>
```

The idea is to modify `ProcessorsFactory` to be able to return processors for a given group name, defaulting on global processors (specified by `preProcessors` and `postProcessors`) if no specific configuration is defined for that group name.

This might require some heavy modifications in wro4j-core (properties parsing, bootstraping, ...) so I want to have your advice before starting this and submitting a pull request.

Thanks.

Apologies for the wall of text :)
